src/components/Notice/NoticeItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+
+import { styled } from 'styled-components';
+
+import Notice from '../../types/Notice';
+import NoticeBottom from './NoticeBottom';
+import NoticeTitle from './NoticeTitle';
+
+const NoticeBox = styled.div`
+padding: 2.3rem;
+padding-inline: ${(props) => props.theme.sizes.contentPadding};
+`;
+
+const NoticeTop = styled.div`
+  display: flex;
+  cursor: pointer;
+`;
+
+const NoticeExtend = styled.div`
+    margin: auto 0;
+`;
+
+type NoticeItemProps = {
+  notice: Notice;
+}
+
+export default function NoticeItem({ notice } : NoticeItemProps) {
+  const [isVisible, setVisible] = useState(false);
+
+  const toggleContent = () => {
+    setVisible(!isVisible);
+  };
```

Comment:
이전 상태값을 사용하는 경우에는 updater function을 사용해주는게 좋아요~
```suggestion
  const toggleContent = () => {
    setVisible(prev => !prev);
  };
```